client/my-sites/checkout/checkout/credit-card-payment-box.jsx

```diff
@@ -179,6 +182,57 @@ export class CreditCardPaymentBox extends React.Component {
 		this.props.onSubmit( event );
 	};
 
+	getPlanProducts() {
+		return this.props.cart.products
+			? this.props.cart.products.filter( ( { product_slug } ) => getPlan( product_slug ) )
+			: false;
+	}
+
+	shouldRenderInstallmentsPlanPicker() {
+		const { cart, transaction } = this.props;
+		if ( typeof transaction.payment === 'undefined' ) {
+			return false;
+		}
+
+		const planInCart = this.getPlanProducts()[ 0 ];
```

Comment:
Is there ever a condition under which `getPlanProducts()` will return false? Maybe we should normalize the return type of `getPlanProducts`, i.e., (`: [];`) to make things more predictable.

Lodash will also do that for us.

```
filter( undefined, ( { product_slug } ) => getPlan( product_slug ) );
// returns `[]`
```